src/methods_segmentation/watershed/config.vsh.yaml

```diff
@@ -0,0 +1,179 @@
+name: watershed
+label: "watershed Segmentation"
+summary: "Output of the segmantation methot watershed"
+description: "Output of the segmantation methot watershed"
+links:
+  documentation: "https://github.com/openproblems-bio/task_ist_preprocessing"
+  repository: "https://github.com/openproblems-bio/task_ist_preprocessing"
```

Comment:
we provide watershed with the txsim repo. Therefore the method should be pointing to txsim. 
As the txsim package doesn't have great documentation you can leave the task_ist_preprocessing repo for documentation, because there we have at least the auto generated one from the yaml:)
On the other hand I must say it's not ideal that we don't have descriptions for the hyper parameters here. Probably there is a label in the viash config to provide those, ... but yea, especially for watershed in would be quite an effort^^